### DISCUSSION TITLE
LMS not properly registered with GitHub Classroom

### DISCUSSION THREAD

**Discussion by suss-userx** [community/community#43281] [GitHub Education]
Hi there, 

I followed [these instructions](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/register-a-learning-management-system-with-github-classroom) to integrate github classroom with Canvas. My team has ensured that each step is completed correctly. 

However, I am still met with the following error `LMS not properly registered with GitHub Classroom` (See screenshot 1) when I follow the suggested instruction to set the LMS, it says that LMS is already registered. (See screenshot 2)

Do you have any suggestion? Thank you

![CleanShot 2023-01-04 at 13 21 35](https://user-images.githubusercontent.com/111267672/210489569-0e48698d-57b6-4d6a-be30-4dbfdddf91ec.jpg)

![CleanShot 2023-01-04 at 13 21 07@2x](https://user-images.githubusercontent.com/111267672/210489482-e7bb9899-4fde-4a08-8972-deb40dd09ef7.jpg)

**Comment by hectarek** [comment]
I also have the same issue but haven't seen anything on this specific issue. I would appreciate anyone's insight on whether it's just how we configured it or if it's something on GitHub's side of things.

**Reply by patchin** [reply]
I have the same issue. Anyone else I can contact for support?

**Reply by mariomack** [reply]
Hi,
I'm having this same issue with Moodle. We've followed all steps but we received the same message "LMS already registered".
Is there a way to "unregister" a LMS?
Any help with this?
Thanks

**Comment by Victor-Morel** [comment]
Hi, similar issue on my side:
we carefully followed the instructions to integrate Classrooms to Canvas, yet the course page on Canvas yields a _{"error":"Invalid Request"}_, and GitHub returns a _LMS Already Registered_, on both sides without any further information.

Any idea on where to look to further the investigation?

Is it possible to unregister an LMS then to register it again?

Best

**Comment by patchin** [comment]
Are you using an on-premise version of Canvas or cloud Canvas? You may want to check that lti_iss (in config/security.yml) equals "https://canvas.instructure.com". You'll have to restart your Canvas app servers after making the change.
